tao_compiler/mlir/disc/BUILD

```diff
@@ -1761,6 +1767,7 @@ transitive_hdrs(
         ":disc_map_chlo_to_hlo_op",
         ":mhlo_disc",
         "//tensorflow/compiler/mlir/hlo:hlo",
+        "//tensorflow/compiler/mlir/hlo:lhlo",
```

Comment:
Why should we expose lmhlo header files?

Reply:
+1

Reply:
Deleted